Logger/Logger.csproj

```
@@ -2,7 +2,8 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <LangVersion>10.0</LangVersion>
+    <LangVersion>12.0</LangVersion>
+	<EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
```

Not sure if you enabled them another way but you could also add these here
```suggestion
	<EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
	<Nullable>enable</Nullable>
	<TreatWarningsAsErrors>true</TreatWarningsAsErrors>
         <EnableNETAnalyzers>true</EnableNETAnalyzers>
         <CodeAnalysisTreatWarningsAsErrors>true</CodeAnalysisTreatWarningsAsErrors>
         <TargetFramework>net8.0</TargetFramework>
```

---

Thanks!